src/main/java/org/folio/inventory/resources/Items.java

```diff
@@ -93,16 +95,12 @@ public void register(Router router) {
     router.put(RELATIVE_ITEMS_PATH_ID).handler(this::update);
     router.delete(RELATIVE_ITEMS_PATH_ID).handler(this::deleteById);
 
-    router.post(RELATIVE_ITEMS_PATH_ID + "/mark-in-process")
-      .handler(handle(this::markAsInProcess));
-    router.post(RELATIVE_ITEMS_PATH_ID + "/mark-in-process-non-requestable")
-      .handler(handle(this::markAsInProcessNonRequestable));
-    router.post(RELATIVE_ITEMS_PATH_ID + "/mark-intellectual-item")
-      .handler(handle(this::markAsIntellectualItem));
-    router.post(RELATIVE_ITEMS_PATH_ID + "/mark-long-missing")
-      .handler(handle(this::markAsLongMissing));
-    router.post(RELATIVE_ITEMS_PATH_ID + "/mark-missing")
-      .handler(handle(this::markAsMissing));
+    Arrays.stream(ItemStatusName.values()).map(itemStatusName -> ItemStatusURL.getUrlForItemStatusName(itemStatusName))
+      .filter(itemStatusUrl -> itemStatusUrl.isPresent())
+      .forEach(itemStatusUrl -> registerMarkItemAsHandler(itemStatusUrl.get(), router));
+
+//    router.post(RELATIVE_ITEMS_PATH_ID + "/mark-missing")
+//      .handler(handle(this::markAsMissing));
```

Comment:
If this is now implemented in the new way, please remove the code rather than commenting it out